test/cpp/jit/test_gpu.cpp

```diff
@@ -754,6 +767,76 @@ void testGPU_FusionReplaceAll() {
   TORCH_CHECK(static_cast<Float*>(bop->lhs())->sameAs(new Float{2.f}));
 }
 
+void testGPU_FusionDependency() {
```

Comment:
NITPICK: strangely some tests have been reordered...

Reply:
I was trying to put things in the order they're listed in tests.h. I don't even remember how far I got before I gave up.